scripts/provisioning/command_line.py

```diff
@@ -0,0 +1,239 @@
+# Copyright 2019, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Command-line sample that creates STS Job Pool and syncs with SDRS.
+
+Note the start times are in UTC (GMT-7 for PDT months)
+Example command line arguments from the prompt:
+'python command_line.py create sdrs-server 2019/05/15 ds-dev-rpo ds-bucket-dev http://localhost:8080/stsjobpool/'
+For more information, see the README.md.
+"""
+
+import argparse
+import datetime
+import json
+import logging
+import requests
+import sys
+import utils
+
+import googleapiclient.discovery
+
+from google.cloud import storage
+
+
+logging.basicConfig()
```

Comment:
basicConfig() uses a SteramHandler, which by default is sys.stderr. So LOGGER.info will go to stderr, not stdout. If the intention is to use LOGGER to just log errors, that'd be fine.